current/README.md

```diff
@@ -12,7 +12,11 @@ Our documentation is divided into several main sections, which cover the Archive
 
 ### Getting Started
 
-- [Installing the Archives Unleashed Toolkit](https://github.com/archivesunleashed/aut-docs-new/blob/master/current/install.md)
+- [Setting up the Archives Unleashed Toolkit](setting-up-aut.md)
+- [Using the Archives Unleashed Toolkit at Scale](aut-at-scale.md)
+- [Archives Unleashed Toolkit Lesson](toolkit-lesson.md)
+- [Docker Installation](docker-install.md)
```

Comment:
This might be best nested underneath the lesson? Since you only really need Docker if you're doing the lesson (it seems to scare people sometimes, and then they run into memory issues once they try to use Docker on their own biggish datasets).